File: src/main/java/com/cartobucket/auth/routes/mappers/ApplicationRequestFilterMapper.java
package com.cartobucket.auth.routes.mappers;

import com.cartobucket.auth.model.generated.ApplicationRequestFilter;

import java.util.List;
import java.util.UUID;

public class ApplicationRequestFilterMapper {
    public static ApplicationRequestFilter to(List<UUID> authorizationServerIds) {
        var filter = new ApplicationRequestFilter();
        filter.setAuthorizationServerIds(authorizationServerIds);
        return filter;
    }
}

File: auth-api/src/main/kotlin/com/cartobucket/auth/api/server/routes/mappers/ApplicationRequestFilterMapper.kt
// (C)2024
package com.cartobucket.auth.api.server.routes.mappers

import com.cartobucket.auth.api.dto.ApplicationRequestFilter
import java.util.UUID

object ApplicationRequestFilterMapper {
    fun to(authorizationServerIds: List<UUID>): ApplicationRequestFilter =
        ApplicationRequestFilter(
            authorizationServerIds = authorizationServerIds,
        )
}
